docs/dev/setup/client.rst

```diff
@@ -31,7 +31,7 @@ command:
 
 .. code:: bash
 
-   npm run serve
+   npm run start --
```

Comment:
Locally, I can use `npm start`. Does this always work? If yes, it would be easier.
Are the `--` required?

Reply:
No, they are not. The `--` are used by npm for option passthrough. I removed them.